pages.es/linux/rpi-otp-private-key.md

```diff
@@ -0,0 +1,8 @@
+# rpi-otp-private-key
+
+> Muestra la clave privada programable una sola vez (OTP) de una Raspberry Pi.
```

Comment:
```suggestion
> Muestra la llave privada de  la clave de un solo uso (OTP) de una Raspberry Pi.
```

Reply:
@tricantivu  there is a double space between `de` and `la`. Please fix it and will commit your suggestion.

Reply:
Done